server/resp/src/main/java/org/infinispan/server/resp/commands/connection/COMMAND.java

```diff
@@ -59,5 +59,7 @@ private void addCommand(StringBuilder builder, RespCommand command) {
       builder.append(":").append(command.getLastKeyPos()).append(CRLF);
       // Step
       builder.append(":").append(command.getSteps()).append(CRLF);
+
+      // TODO use RespCommand#describeSubCommands.
```

Comment:
Looks like something is missing?

Reply:
Reverted that. The `COMMAND` has a section for sub-commands, but that requires additional stuff.